iiwa_front:
  striker_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - F_striker_joint_1
      - F_striker_joint_2

    F_striker_joint_1:
      pid: { p: 100.0, i: 1.0, d: 5.0, i_clamp_min: -10000, i_clamp_max: 10000 }
    F_striker_joint_2:
      pid: { p: 100.0, i: 1.0, d: 5.0, i_clamp_min: -10000, i_clamp_max: 10000 }

iiwa_back:
  striker_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - B_striker_joint_1
      - B_striker_joint_2

    B_striker_joint_1:
      pid: { p: 100.0, i: 0.1, d: 5.0, i_clamp_min: -10000, i_clamp_max: 10000 }
    B_striker_joint_2:
      pid: { p: 100.0, i: 0.1, d: 5.0, i_clamp_min: -10000, i_clamp_max: 10000 }